examples/tvhc/tvhc.cpp

```diff
@@ -849,7 +849,7 @@ void recordTopicDefinitions( TTopicDefinition *p, THelpFile& helpFile )
 {
     while (p != 0)
         {
-        resolveReference(p->topic, p->value, *(helpFile.stream));
+        resolveReference(p->topic, p->value, dynamic_cast<fpstream&>(*(helpFile.stream)));

```

Comment:
Good call, a better fix would been to have to plumb the fpstream -> iopstream all the way through instead of stopping here. In tvhc this is always an fpstream, but I think your suggestion is better.